proto/zitadel/resources/object/v3alpha/object.proto

```diff
@@ -17,27 +17,68 @@ message Details {
       example: "\"69629012906488334\"";
     }
   ];
+  //the timestamp of the first event applied to the object.
+  google.protobuf.Timestamp created = 3;
+  //the timestamp of the last event applied to the object.
+  google.protobuf.Timestamp changed = 4;
+  //the parent object representing the returned objects context.
+  optional zitadel.object.v3alpha.Owner owner = 5 [
```

Comment:
why is this optional?

Reply:
This should only be returned where it adds information, like if you query org context resources instance-wide

Reply:
I discussed it with @stebenz and we decided to always return the owner again.
I removed the word optional.